docs/01-module-block-1/02-learning-goals.adoc

```diff
@@ -1,25 +1,44 @@
 === {learning-goals}
 
 // tag::DE[]
-Die Teilnehmer:Innen …
+In diesem Kapitel erreichen die Teilnehmer:innen die folgenden Lernziele:
 
 [[LZ-1-1]]
-==== LZ 1-1: Wissen wie man Künstliche Intelligenz definiert und die Anordnung zu Machine Learning, Data Science, Deep Learning, Generative AI.
+==== LZ 1-1: Definition Künstliche Intelligenz inklusive der Einordnung von Machine Learning, Data Science, Deep Learning, Generative AI
+
+Die Teilnehmer:innen wissen wie man Künstliche Intelligenz definiert und wie Machine Learning, Data Science, Deep Learning, Generative AI darin eingeordnet werden.
 
 [[LZ-1-2]]
-==== LZ 1-2: Wissen, wie man typische allgemeine Use-Cases für KI spezifiziert, z.B. Bilderkennung & -erzeugung, Sprachverarbeitung, Vorhersagen, Personalisierung und Anomalieerkennung.
+==== LZ 1-2: Spezifikation typischer allgemeiner Anwendungsfälle für KI
+
+Die Teilnehmer:innen wissen wie typische allgemeine Anwendungsfälle für KI spezifiziert werden können. Dies umfasst Anwundungsfälle für  z.B. Bilderkennung & -erzeugung, Sprachverarbeitung, Vorhersagen, Personalisierung und Anomalieerkennung.
 
 [[LZ-1-3]]
-==== LZ 1-3: Kennen Einsatzmöglichkeiten in verschiedenen Branchen, z.B. Marketing, Medizin, Robotik und Content-Creation.
+==== LZ 1-3: Einsatzmöglichkeiten in verschiedenen Branchen
```

Comment:
Hier wäre auch ein Verb noch hilfreich, also z.B.
> Einsatzmöglichkeiten in verschiedenen Branchen kennen

In dem Fall ist es sinnvoll und vollkommen ok, "kennen" zu verwenden.